src/modules/components/proposal/proposalActions/proposalActions/proposalActions.test.tsx

```diff
@@ -110,4 +112,19 @@ describe('<ProposalActions /> component', () => {
 
         expect(screen.getByText(`Custom action for ${actions[1].type}`)).toBeInTheDocument();
     });
+
+    it('renders an empty state if no actions are provided', () => {
+        const emptyStateDescription = 'Custom empty state description';
+        const actions: IProposalAction[] = [];
+        render(createTestComponent({ actions, emptyStateDescription }));
+        expect(screen.getByText(modulesCopy.proposalActionsContainer.empty.heading)).toBeInTheDocument();
+        expect(screen.getByText(emptyStateDescription)).toBeInTheDocument();
+    });
```

Comment:
We're asserting the same things on both tests otherwise
```suggestion
    it('renders an empty state if no actions are provided', () => {
        const actions: IProposalAction[] = [];
        render(createTestComponent({ actions }));
        expect(screen.getByText(modulesCopy.proposalActionsContainer.empty.heading)).toBeInTheDocument();
    });
```